examples-standalone/electron-dashboard/src/app/home/home.component.html

```diff
@@ -1,7 +1,33 @@
 <div class="container">
-  <h1 class="title">
-    {{ 'PAGES.HOME.TITLE' | translate }}
-  </h1>
-
-  <a routerLink="/detail">{{ 'PAGES.HOME.GO_TO_DETAIL' | translate }}</a>
+  <div class="banner">
+    <h1>Kendo UI for Angular</h1>
+    <p>
+      Build responsive and powerful web applications with Kendo UI for Angular.
+    </p>
+    <a
+      href="https://www.telerik.com/try/kendo-angular-ui"
+      class="try-button"
+      target="_blank"
+      >Try it out</a
+    >
+  </div>
+  <kendo-grid
+    [style.width.%]="50"
+    [kendoGridBinding]="gridData"
+    [pageSize]="10"
+    [pageable]="true"
+    [sortable]="true"
+    [filterable]="true"
+    [groupable]="true"
+    [height]="420"ß
```

Comment:
There is a ß that could be removed.

I would suggest increasing the height so that the scroll disappears.